app/src/main/java/org/merakilearn/datasource/network/SaralApi.kt

```diff
@@ -43,4 +43,9 @@ interface SaralApi {
     @GET("users/EnrolledBatches")
     suspend fun getEnrolledBatches():Response<List<Batches>>
 
+    @GET("partners/{partnerID}")
+    suspend fun getPartnerData(
+        @Path(value = "partnerID") partnerID: Int?
```

Comment:
Let's not make this partnerId parameter nullable.
Its should be just this:-
suspend fun getPartnerData(partnerId: Int)